tests/Tests/100__deploy/100__installation/103__grafana.robot

```diff
@@ -0,0 +1,76 @@
+*** Settings ***
+Documentation       Post install test cases that verify OCP Grafana resources and objects
+
+Library             OpenShiftLibrary
+Resource            ../../../Resources/ODS.robot
+
+*** Test Cases ***
+Verify Grafana Is Shipped And Enabled Within ODS
+    [Documentation]    Verify Grafana Is Shipped And Enabled Within ODS
+    [Tags]    Sanity
+    ...       Smoke
+    ...       Tier1
+    ...       ODS-231
+    @{grafana_pods} =    Fetch Grafana Pods
+    @{grafana_deployment} =    Fetch Grafana Deployments
+    @{grafana_services} =    Fetch Grafana Services
+    @{grafana_routes} =    Fetch Grafana Routes
+    @{grafana_replicasets} =    Fetch Grafana ReplicaSets
+    Verify Resources Values    status.phase    Running    @{grafana_pods}
+    Verify Resources Values    status.containerStatuses[0].name    auth-proxy    @{grafana_pods}
+    Verify Resources Values    status.containerStatuses[1].name    grafana    @{grafana_pods}
+    Verify Resources Values    status.conditions[2].status    True    @{grafana_pods}
+    Verify Resources Values    status.replicas    2    @{grafana_deployment}
+    Verify Resources Values    status.readyReplicas    2    @{grafana_deployment}
+    Verify Resources Values    spec.ports[0].name    https    @{grafana_services}
+    Verify Resources Values    spec.ports[0].port    443    @{grafana_services}
+    Verify Resources Values    spec.ports[0].protocol    TCP    @{grafana_services}
+    Verify Resources Values    spec.ports[0].targetPort    8443    @{grafana_services}
+    Verify Resources Values Using RegExp    spec.clusterIP    ^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])(\.(?!$)|$)){4}$
+    ...    @{grafana_services}
+    Verify Resources Values    status.readyReplicas    2    @{grafana_replicasets}
+    Verify Resources Values    status.replicas    2    @{grafana_replicasets}
+    Verify Resources Values    spec.port.targetPort    https    @{grafana_routes}
+    Verify Resources Values    spec.to.name    grafana    @{grafana_routes}
+    Verify Resources Values Using RegExp    spec.host    ^(grafana-redhat-ods-monitoring.*)    @{grafana_routes}
+
+
+*** Keywords ***
+Fetch Grafana Pods
+    [Documentation]    () -> list(dict) 
```

Comment:
hey @pablofelix  could u plz elaborate Documentation  
```() -> list(dict)```---> is bit difficult to understand

Reply:
This is the sort notation with the function's input and output for a quick overview of the function. In order to further explain the function, I also included the long version in the documentation. I didn't write the arguments because I thought that was verbose but do you think it would be better to add args None in the documentation?
Args:
   None

Reply:
ack

Reply:
done

Reply:
You could think this keyword is returning a list of pods, but I think its returning all resource information related to the grafana pods

What about calling it  `Fetch Grafrana Pod Resources` or  `Fetch Grafrana Pod Info`?

I also agree with Milind that the documentation is a little bit confusing. I think a plain text will be easier to understand
```
Fetch Grafrana Pod Info
[Documentation] Returns the resource information for the grafana pods in ODS
...    Args:
...        None
...    Returns:
...        grafana_pods(list(dict)): Grafana pod information selected by label and namespace   
```